### PR TITLE
[FE]: Implement Landing Page Screen and tailwind color added

### PR DESCRIPTION
**Pull request by robertram**
# Add Landing Page Component for StarShop

## Description
This PR introduces a new landing page component that serves as the main entry point for StarShop. 

### Key Features:
- Responsive layout that adapts to mobile, tablet, and desktop views
- Hero section with marketplace value proposition
- "Register Now" CTA button
- Key statistics display
- StarShop logo

### Technical Details:
- Built using React/Next.js with TypeScript
- Utilizes Tailwind CSS for responsive styling
- Implements Next.js Image component for optimized image loading
- Mobile-first design with breakpoints for md (768px) and xl (1280px)

### TODO:
- [x] Implement header section (currently placeholder)
- [X] Add analytics tracking for CTA button
- [X] Consider A/B testing different headline variations

## Testing
Please verify:
- Responsive behavior across different screen sizes
- Image loading and optimization
- Link functionality to registration page
- Text readability and contrast ratios

## Screenshots

Desktop
Mobile

### CODE REVIEW

File: src/components/landing-page/LandingComponent.tsx [hunk 1]
```diff
@@ -1,9 +1,55 @@
...
+        </div>
+        <div className='mx-auto xl:m-0 w-[400px] md:w-1/2 flex justify-center items-center px-4 my-[0px] '>
+          <Image
+            src={StarShopLanding}
+            alt="StarShop Logo"
+            width={700}
+            height={20}
+          />
+        </div>
+      </div>
```
Comment (KevinLatino) [inline src/components/landing-page/LandingComponent.tsx:52]:
```suggestion
      <div className='text-white h-[120px]'>
        Add header here
      </div>

      <div className='p-0 flex flex-col-reverse md:flex-row w-full xl:my-[35px] xl:pl-[100px] px-[10px] md:px-[50px] pb-[100px] text-white max-w-[1600px] m-auto'>
        <div className='w-full md:w-1/2 xl:max-w-[700px]'>
          <h1 className='uppercase text-white text-[50px] md:text-[64px] font-extrabold leading-[50px] md:leading-[64px]'>
            Marketplace built on stellar blockchain
          </h1>
          <p className='text-white text-lg sm:text-sm mt-[20px] md:mt-[35px]'>
            StarShop enables small businesses to sell products transparently while fostering trust. Customers receive unique NFTs as digital collectibles with each purchase, and businesses earn milestone NFTs to showcase their success.
          </p>
          <Link
            href="/register"
            className='mt-[30px] md:mt-[50px] font-bold inline-block bg-white text-primary-purple px-[10px] md:px-[40px] py-[15px] rounded-[25px] hover:bg-purple-100 leading-[22.72px] text-left cursor-pointer'
          >
            Register Now
          </Link>
          <div className='flex flex-col md:flex-row mt-[50px] !text-center'>
            <div className='mb-6'>
              <div className="text-4xl sm:text-4xl font-semibold md:mb-2">200+</div>
              <div className="text-lg sm:text-sm">Sellers</div>
            </div>
            <div className='h-16 min-w-[2px] bg-white mx-[50px] hidden md:block'></div>
            <div className='mb-6'>
              <div className="text-4xl sm:text-4xl font-semibold md:mb-2">2000+</div>
              <div className="text-lg sm:text-sm">High-Quality<br className="hidden sm:block" /> products</div>
            </div>
            <div className='h-16 min-w-[2px] bg-white mx-[50px] hidden md:block'></div>
            <div>
              <div className="text-4xl sm:text-4xl font-semibold md:mb-2">30,000+</div>
              <div className="text-lg sm:text-sm">Happy Customers</div>
            </div>
          </div>
        </div>
        <div className='mx-auto xl:m-0 w-[400px] md:w-1/2 flex justify-center items-center px-4 my-[0px] '>
          <Image
            src={StarShopLanding}
            alt="StarShop Logo"
            width={700}
            height={20}
          />
        </div>
      </div>
```

Please implement semantic HTML for SEO improvement:      

<main className='p-0 flex flex-col-reverse md:flex-row w-full xl:my-[35px] xl:pl-[100px] px-[10px] md:px-[50px] pb-[100px] text-white max-w-[1600px] m-auto'>
        <section className='w-full md:w-1/2 xl:max-w-[700px]'>
          <h1 className='uppercase text-white text-[50px] md:text-[64px] font-extrabold leading-[50px] md:leading-[64px]'>
            Marketplace built on stellar blockchain
          </h1>
          <p className='text-white text-lg sm:text-sm mt-[20px] md:mt-[35px]'>
            StarShop enables small businesses to sell products transparently while fostering trust. Customers receive unique NFTs as digital collectibles with each purchase, and businesses earn milestone NFTs to showcase their success.
          </p>
          <Link
            href="/register"–
            className='mt-[30px] md:mt-[50px] font-bold inline-block bg-white text-primary-purple px-[10px] md:px-[40px] py-[15px] rounded-[25px] hover:bg-purple-100 leading-[22.72px] text-left cursor-pointer'
          >
            Register Now
          </Link>

          <div className='flex flex-col md:flex-row mt-[50px] !text-center'>
            <article className='mb-6'>
              <h2 className="text-4xl sm:text-4xl font-semibold md:mb-2">200+</h2>
              <p className="text-lg sm:text-sm">Sellers</p>
            </article>
            <div className='h-16 min-w-[2px] bg-white mx-[50px] hidden md:block' aria-hidden="true"></div>
            <article className='mb-6'>
              <h2 className="text-4xl sm:text-4xl font-semibold md:mb-2">2000+</h2>
              <p className="text-lg sm:text-sm">High-Quality<br className="hidden sm:block" /> products</p>
            </article>
            <div className='h-16 min-w-[2px] bg-white mx-[50px] hidden md:block' aria-hidden="true"></div>
            <article>
              <h2 className="text-4xl sm:text-4xl font-semibold md:mb-2">30,000+</h2>
              <p className="text-lg sm:text-sm">Happy Customers</p>
            </article>
          </div>
        </section>

        <section className='mx-auto xl:m-0 w-[400px] md:w-1/2 flex justify-center items-center px-4 my-[0px]'>
          <Image
            src={StarShopLanding}
            alt="StarShop Logo"
            width={700}
            height={20}
          />
        </section>
      </main>

File: src/components/landing-page/LandingComponent.tsx [hunk 2]
```diff
@@ -1,9 +1,55 @@
...
+import Link from 'next/link'
+import Image from "next/image"
 
 const LandingPageComponent: React.FC = () => {
   return (
     <>
-      <h1 className='text-white text-2xl font-bold mt-8'>Landing page</h1>
+      <div className='text-white h-[120px]'>
+        Add header here
+      </div>
```
Comment (KevinLatino) [inline src/components/landing-page/LandingComponent.tsx:11]:
```suggestion
      <div className='text-white h-[120px]'>
        Add header here
      </div>
```
Delete this part of the code please

**Review by KevinLatino — COMMENTED** [review]
Good Job! Here are some comments:

File: src/components/landing-page/LandingComponent.tsx (hunk 2 above)
Comment (robertram) [reply src/components/landing-page/LandingComponent.tsx]: Fixed!

File: src/components/landing-page/LandingComponent.tsx (hunk 1 above)
Comment (robertram) [reply src/components/landing-page/LandingComponent.tsx]: Fixed!

**Pull request merged**